packages/devtools_app/lib/src/network/network_request_inspector_views.dart

```diff
@@ -136,6 +136,22 @@ class HttpRequestHeadersView extends StatelessWidget {
   }
 }
 
+class HttpResponseView extends StatelessWidget {
+  const HttpResponseView(this.data);
+
+  final HttpRequestData data;
+
+  @override
+  Widget build(BuildContext context) {
+    return SingleChildScrollView(
+      child: Text(
+        data.responseBody,
+        style: const TextStyle(fontFamily: 'RobotoMono'),
```

Comment:
also move this monospace font to a theme.

Reply:
used `fixedFontStyle` from theme.dart